Source/doomtype.h

```diff
@@ -84,47 +89,39 @@ typedef int64_t Long64;
 #define inline __inline
 #endif
 
-// The packed attribute forces structures to be packed into the minimum
-// space necessary.  If this is not done, the compiler may align structure
-// fields differently to optimize memory access, inflating the overall
-// structure size.  It is important to use the packed attribute on certain
-// structures where alignment is important, particularly data read/written
-// to disk.
-
-#ifdef __GNUC__
- #if defined(_WIN32) && !defined(__clang__)
-  #define PACKEDATTR __attribute__((packed,gcc_struct))
- #else
-  #define PACKEDATTR __attribute__((packed))
- #endif
-
+#if defined(__GNUC__)
  #define PRINTF_ATTR(fmt, first) __attribute__((format(printf, fmt, first)))
  #define PRINTF_ARG_ATTR(x) __attribute__((format_arg(x)))
  #define NORETURN __attribute__((noreturn))
 #else
- #if defined(_MSC_VER)
-  #define PACKEDATTR __pragma(pack(pop))
- #else
-  #define PACKEDATTR
- #endif
-
  #define PRINTF_ATTR(fmt, first)
  #define PRINTF_ARG_ATTR(x)
  #define NORETURN
 #endif
 
-#ifdef __WATCOMC__
- #define PACKEDPREFIX _Packed
-#elif defined(_MSC_VER)
- #define PACKEDPREFIX __pragma(pack(push,1))
+// The packed attribute forces structures to be packed into the minimum
+// space necessary.  If this is not done, the compiler may align structure
+// fields differently to optimize memory access, inflating the overall
+// structure size.  It is important to use the packed attribute on certain
+// structures where alignment is important, particularly data read/written
+// to disk.
+
+#if defined(__GNUC__)
```

Comment:
Sorry, but since this code gets rewritten anyway: Could you rename the macros to PACKED_PREFIX and PACKED_SUFFIX, respectively, please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/400/commits/5664324542133db5f7ec473175337dc9a8d22ca8